Use account-level SQL traits to associate users to the account (#5604)
segmentio/segment-docs: src/engage/audiences/account-audiences.md

> info ""
> Use SQL traits for complex calculations not supported by computed traits. For example, you would use SQL traits to calculate the number of unique users associated with an account who have logged in during the past month.

> ### Use account-level SQL traits to associate users to the account

Reviewer: ```suggestion
### Use account-level SQL traits to associate users to accounts
```

> To associate users to an account via SQL traits, we have to return both group_id and user_id in the account level SQL trait - this will fire a group call with the userId and the groupId in question, which is the method that Segment uses to add users to groups in destinations.

Reviewer: ```suggestion
To associate users to an account with SQL traits, you must return both the `group_id` and `user_id` in the account level SQL trait. This fires a Group call which Segment uses to add users to groups in destinations.
```

> There is a limitation on Segment end when a group (account) contains more than one user, the query will return duplicate group_ids (mapped to unique user_ids). Segment won't allow us to return duplicate group_id's in the Group(account level) SQL trait., So we can't map the users to the accounts on many-to-many situation.

Reviewer: ```suggestion
When a group (account) contains more than one user, the query returns duplicate `group_id`s (mapped to unique `user_id`s). However, Segment doesn't return duplicate `group_id`s in the group (account level) SQL trait. As a result, you can't map the users to accounts in a many-to-many situation.
```
